.editorconfig

```diff
@@ -0,0 +1,14 @@
+root = true
+
+[*]
+indent_style = tab
```

Comment:
We are actually using smart tabs, not just tabs, but not sure what's the current best practice to specify this in `.editorconfig`. Relevant: https://github.com/editorconfig/editorconfig/issues/323

Reply:
Hmm, I'm not familiar with smart tabs. 🤔 
I'll look into it and see how that works with EditorConfig, sorry about that.

Reply:
Well, some people say that smart tabs is just how tabs *should* be used anyway (as you will see in that discussion as well). Basically, tabs for indentation and spaces for alignment. That way everyone gets to have their preferred tab width, and things that should be aligned, stay aligned.